lms/static/sass/shared/_footer.scss

```diff
@@ -488,4 +510,9 @@ $edx-footer-bg-color: rgb(252,252,252);
       width: 129px;
     }
   }
+
+  @media print {
```

Comment:
this should also be moved to the top of the list of styles being added to this level of depth in the SASS, making it clearer what its parent container is.